src/buckets/actions/thunks.ts

```diff
@@ -344,6 +363,21 @@ export const addSchemaToBucket = (
   }
 }
 
+export const addSchemaToBucket = (
+  bucketID: string,
+  orgID: string,
+  bucketName: string,
+  schema: typeof MeasurementSchemaCreateRequest
+) => async (dispatch: Dispatch<Action>) => {
+  await addMeasurementSchemaToBucketInternal(
```

Comment:
does this need to be `await`ed if there's nothing below this code doing anything with the result?

is this equivalent to

```ts
addMeasurementSchemaToBucketInternal(
  bucketID,
  schema,
  orgID,
  bucketName,
  dispatch
).then()
```
?

Reply:
hmm...I thought I needed the await when i did the refactor.  turns out I don't.  removing it.